README.md

```diff
@@ -4,13 +4,14 @@ Code to run on every board, to run canary tests
 ## To set up a new board:
 - Create a new robot, likely in the `viam-hardware-testing` org, in the "Board Canaries" location.
 - Add a board to that robot, whose type matches your physical hardware.
-- In the `Auth/Network` tab of the config set the bind address to port 9090.
+- In the `CONFIGURE -> Builder` tab click the `+` sign and select `Network`. Then set the bind address to port 9090 in `Network`.
 - Install Viam on the physical board, per the Setup tab in the robot you just created. **However,** put the config file in `/etc/viam-canary.json` instead of `/etc/viam.json`.
 - Jump together 2 pins on your board. Ideally, one of them (the "output" pin) will also have hardware PWM enabled.
   - If you have a board whose GPIO pins cannot also do hardware PWM but you want to test hardware PWM separately, jump together a second set of pins. One set will be to test GPIO input and output, and the other will test hardware PWM output and digital interrupt input.
   - Also jump together 2 more gpio pins to test software PWM.
 - Clone this repo onto the board.
 - Run `sudo ./install.sh`
+  - if you are getting `error: externally-managed-environment` while running the command, then you need to setup `venv` in the directory. 
```

Comment:
not sure if this is necessary, I don't feel strongly about this line.
I can remove it or add some more instruction around setting up venv. @penguinland

Reply:
I'm working on a branch to do venv stuff automatically. When it's out, let's discuss.

Reply:
See https://github.com/viamrobotics/board_canaries/pull/39. Hopefully this line is no longer needed.

Reply:
Yeah not needed. I'll remove it